commerce/endpoint/account/address/classes/AddressExtensionProviderSample.cls

```diff
@@ -0,0 +1,19 @@
+// This corresponds to /commerce/webstores/${webstoreId}/accounts/${accountId}/addresses/${addressId} and Commerce_Endpoint_Account_Address
```

Comment:
Is there a reason for using single line comments instead of "javadoc" style comments for classes and methods?
```suggestion
/**
  *This corresponds to /commerce/webstores/${webstoreId}/accounts/${accountId}/addresses/${addressId} and Commerce_Endpoint_Account_Address
  */
```